tests/spec/puppeteer-spec.js

```diff
@@ -0,0 +1,98 @@
+jest.setTimeout(20000);
+
+describe("An editor for MicroPython running at localhost.", function() {
+
+    beforeAll(async() => {
+        global.browser = await global.puppeteer.launch({headless: true, args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-dev-shm-usage']});
```

Comment:
Could you add a small comment about why these flags are used?

Reply:
Added in https://github.com/microbit-rosslowe/PythonEditor/pull/9/commits/3dcdaf967f1b0b1ec45374b6e936c721b0e0781d